terraform/security_groups.tf

```diff
@@ -168,3 +168,42 @@ resource "aws_security_group_rule" "sandbox_simple_database" {
   to_port                  = 5432
   source_security_group_id = "${aws_security_group.sandbox_simple_server.id}"
 }
+
+
+# Pre-prod security groups for elasticache
+resource "aws_security_group" "pre_prod_simple_elasticache" {
+  name        = "simple-pre-prod-elasticache-sg"
+  description = "Security group for all pre-prod elasticache instances"
+  vpc_id = "${aws_vpc.simple_servers_01.id}"
+}
+
+#
+# Pre-prod security group for elasticache is shared by sbx, qa and staging
+# Hence, there's one rule for each
+#
+resource "aws_security_group_rule" "sandbox_simple_elasticache" {
+  security_group_id        = "${aws_security_group.pre_prod_simple_elasticache.id}"
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 6379
+  to_port                  = 6379
+  source_security_group_id = "${aws_security_group.sandbox_simple_server.id}"
+}
+
+# resource "aws_security_group_rule" "qa_simple_elasticache" {
```

Comment:
We should uncomment and add support for other envs as well.

Reply:
I was thinking of doing that separately since it requires setting up security groups for each, but I'll do that.